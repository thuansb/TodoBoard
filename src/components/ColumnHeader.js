import React from 'react';
import CounterBox from './CounterBox';

function ColumnHeader({ title, count }) {
    return (
        <div className="column__header">
            <div className="column__header__title">
                {title}
            </div>
            <div className="column__header__counter">
                <CounterBox count={count} />
            </div>
        </div>
    )
}

export default ColumnHeader;