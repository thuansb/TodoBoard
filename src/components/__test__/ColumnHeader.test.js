import React from 'react';
import ColumnHeader from '../ColumnHeader';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <ColumnHeader title="TODO" counter={10000} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
