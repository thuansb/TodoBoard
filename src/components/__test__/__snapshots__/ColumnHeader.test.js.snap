// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="column__header"
>
  <div
    className="column__header__title"
  >
    TODO
  </div>
  <div
    className="column__header__counter"
  >
    <div
      className="counter-box"
    >
      <div
        className="counter-box__text"
      >
        TASKS
      </div>
    </div>
  </div>
</div>
`;
